ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js

```diff
@@ -129,6 +131,76 @@ $(document).ready(function () {
        getCountriesList();
     }
 });
+
+const mapUserFields = function (useraddressdetails){
+	const getAddressFilelds = $(".add-addr-feilds");
+    if(useraddressdetails && getAddressFilelds){
+        getAddressFilelds.toArray().forEach((addressfielditems) => {
+                                            switch(addressfielditems.name){
```

Comment:
since all element keys are unique, we can assign the value directly instead of looping

Reply:
Since we r picking some values so it would not be feasible to run loop, so we r using switch case.